bazelisk_version_test.go

```diff
@@ -237,10 +237,10 @@ func buildGCSResponseOrFail(t *testing.T, prefixes []string, items []interface{}
 		Prefixes: prefixes,
 		Items:    items,
 	}
-	if bytes, err := json.Marshal(r); err != nil {
+	byteValue, err := json.Marshal(r)
+	if err != nil {
```

Comment:
What did `go lint` complain about here? 🤔

Reply:
1. The return should not have been indented
2. bytes conflicts with the package import

Reply:
Got it. Thanks for explaining!